src/components/formatters/test/CFormatterJsonBase_test.cc

```diff
@@ -48,54 +50,60 @@ using namespace NsSmartDeviceLink::NsJSONHandler::Formatters;
 TEST(CFormatterJsonBaseTest, JSonStringValueToSmartObj_ExpectSuccessful) {
   // Arrange value
   std::string string_val("test_string");
-  Json::Value json_value(string_val);  // Json value from string
+  utils::json::JsonValue json_value(string_val);
   SmartObject object;
   // Convert json to smart object
-  CFormatterJsonBase::jsonValueToObj(json_value, object);
+  utils::json::JsonValueRef json_value_ref = json_value;
+ 
+  CFormatterJsonBase::jsonValueToObj(json_value_ref, object);
   // Check conversion was successful
   EXPECT_EQ(string_val, object.asString());
 }
 
 TEST(CFormatterJsonBaseTest, JSonDoubleValueToSmartObj_ExpectSuccessful) {
   // Arrange value
   double dval = 3.512;
-  Json::Value json_value(dval);  // Json value from double
+  utils::json::JsonValue json_value(dval);
   SmartObject object;
   // Convert json to smart object
-  CFormatterJsonBase::jsonValueToObj(json_value, object);
+  utils::json::JsonValueRef json_value_ref = json_value;
+  CFormatterJsonBase::jsonValueToObj(json_value_ref, object);
   // Check conversion was successful
   EXPECT_DOUBLE_EQ(dval, object.asDouble());
 }
 
 TEST(CFormatterJsonBaseTest, JSonMinIntValueToSmartObj_ExpectSuccessful) {
   // Arrange value
   Json::Int ival = Json::Value::minInt;
-  Json::Value json_value(ival);  // Json value from possible minimum signed int
+  utils::json::JsonValue json_value(static_cast<long long int>(ival));
```

Comment:
@dev-gh , in this case will be ambiguous function call. Tried uint64_t / int64_t